packages/backend/src/lib/backends/github.sync.ts

```diff
@@ -93,17 +96,79 @@ export async function getInsightFromRepository(owner: string, repo: string): Pro
   const repository = await getRepository(owner, repo);
   logger.debug(JSON.stringify(repository, null, 2));
 
+  const insight: IndexedInsight = {
+    itemType: ItemType.INSIGHT,
+    namespace: repository.owner.login,
+    name: repository.name,
+    fullName: `${repository.owner.login}/${repository.name}`,
+    description: repository.description ?? '',
+    createdAt: repository.createdAt!,
+    updatedAt: repository.updatedAt!,
+    syncedAt: new Date().toISOString(),
+    stars: repository.stargazers!.totalCount,
+    forks: repository.forkCount!,
+    tags: repository.repositoryTopics!.edges.map(({ node }: any) => node.topic.name),
+    contributors: [],
```

Comment:
Moved getting contributers out of `getInsightFromRepository()` into its own function, so this returns `[]` for now.